# Rover Service
# ARM/DISARM motors
bool state
---
int8 result